frontend/src/components/pages/TranslationPage.css

```diff
@@ -0,0 +1,16 @@
+.translation {
```

Comment:
very nice looking progress bar 👌 did you run it by Shaahana?

Reply:
sorry completely slipped my mind just messaged i will make any updates according to her comments